content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,162 @@
+image_facebook: /static/images/experiments/lockbox/social/facebook.jpg
+
+thumbnail: /static/images/experiments/lockbox/icon/thumbnail.png
+gradient_start: '#00BCD8'
+gradient_stop: '#0086DC'
+
+xpi_url: 'https://testpilot.firefox.com/files/experiment-name/latest'
+addon_id: 'experiment-name@mozilla.com'
+
+created: '2016-01-01T00:00:00.000000Z'
+# completed: '2016-01-01T00:00:00.000000Z'
+# uninstalled: '2016-01-02T00:00:00.000000Z'
+# launch_date: '2016-01-01T00:00:00.000000Z'
+
+changelog_url: 'https://github.com/mozilla-lockbox/lockbox-ios/releases'
+contribute_url: 'https://github.com/mozilla-lockbox/lockbox-ios'
+bug_report_url: 'https://github.com/mozilla-lockbox/lockbox-ios/issues'
+discourse_url: 'https://discourse.mozilla.org/c/test-pilot/lockbox'
+privacy_notice_url: 'https://lockbox.firefox.com/privacy.html'
+
+details:
+  -
+    image: /static/images/experiments/lockbox/details/access.png
+    copy: >
+      Access username and password information for all the accounts you save to Firefox.
+
+  -
+    image: /static/images/experiments/lockbox/details/sync.png
+    copy: >
+      Manage accounts in Firefox and sync changes automatically to Firefox Lockbox.
+
+  -
+    image: /static/images/experiments/lockbox/details/sort-filter.png
+    copy: >
+      Sort and filter on your accounts to quickly locate the one you need.
+
+contributors:
+  -
+    display_name: 'Sandy Sage'
+    title: 'Sr. Product Manager'
+    avatar: /static/images/experiments/lockbox/avatars/ssage.png
+  -
+    display_name: 'Devin Reams'
+    title: 'Engineering Manager'
+    avatar: /static/images/experiments/lockbox/avatars/dreams.png
+  -
+    display_name: 'Ryan Gaddis'
+    title: 'Sr. Interaction Designer'
+    avatar: /static/images/experiments/lockbox/avatars/rgaddis.png
+  -
+    display_name: 'Matt Miller'
+    title: 'Engineering Architect'
+    avatar: /static/images/experiments/lockbox/avatars/mmiller.png
+  -
+    display_name: 'Sasha Heinen'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/sheinen.png
+  -
+    display_name: 'Jim Porter'
+    title: 'Sr. Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/jporter.png
+  -
+    display_name: 'James Hugman'
+    title: 'Mobile Engineer'
+    avatar: /static/images/experiments/lockbox/avatars/jhugman.png
+  -
+    display_name: 'Leif Oines'
+    title: 'Product Data Scientist'
+    avatar: /static/images/experiments/lockbox/avatars/loines.png
+  -
+    display_name: 'Heather McGaw'
+    title: 'Sr. UX Researcher'
+    avatar: /static/images/experiments/lockbox/avatars/hmcgaw.png
+  -
+    display_name: 'Michelle Heubusch'
+    title: 'Content Strategy Lead'
+    avatar: /static/images/experiments/lockbox/avatars/mheubusch.png
+  -
+    display_name: 'Nick Brandt'
+    title: 'Sr. UX Designer'
+    avatar: /static/images/experiments/lockbox/avatars/nbrandt.png
+  -
+    display_name: 'Peter deHaan'
```

Comment:
Feel free to remove this 🤡.
I can think of several other QA people far more deserving of credit than this donkey.

Reply:
> can think of several other QA people far more deserving of credit

Agreed. I've added to our checklist above to finalize our contributor list.